### PR TITLE
Fix rvcount

### PR DESCRIPTION
**Pull request by Jim8y**
closes https://github.com/neo-project/neo-devpack-dotnet/issues/1013

### CODE REVIEW

**Comment by Jim8y** [comment]
@shargon this one has solved the problem in the issue, but there is a problem in the compiler i havent located and i will fix in another pr.

File: examples/Example.SmartContract.NFT/Loot.cs [hunk 1]
```diff
@@ -180,10 +180,11 @@ public void OwnerClaim(BigInteger tokenId)
         /// </summary>
         /// <param name="tokenId"></param>
         /// <param name="sender"></param>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        // [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private void MintToken(BigInteger tokenId, UInt160 sender)
         {
-            var credential = CheckClaim(tokenId);
```
Comment (shargon) [inline examples/Example.SmartContract.NFT/Loot.cs:186]: Why it fault?

Comment (Jim8y) [reply examples/Example.SmartContract.NFT/Loot.cs:186]: rv count incorrect. Need more time to debug.

**Comment by Jim8y** [comment]
@Jim8y  please merge.

**Review by shargon — APPROVED** [review]

**Pull request merged**